src/gui/Base/HTile.qml

```diff
@@ -110,12 +110,21 @@ HButton {
 
     TapHandler {
         acceptedButtons: Qt.RightButton
+        acceptedPointerTypes: PointerDevice.GenericPointer
         onTapped: {
             rightClicked()
             if (contextMenu) contextMenuLoader.active = true
         }
     }
 
+    TapHandler {
+        acceptedPointerTypes: PointerDevice.Finger | PointerDevice.Pen
+        onLongPressed: {
+            rightClicked()
+            if (contextMenu) contextMenuLoader.active = true
```

Comment:
About the menu opening in the top left corner, could you try to get the menu opened via [`popup()`](https://doc.qt.io/qt-5/qml-qtquick-controls2-menu.html#popup-method) instead? That may solve it

Reply:
expected error:

```
qrc:/src/gui/Base/HTile.qml:124: TypeError: Property 'popup' of object QQuickLoader(0xb8eb9208) is not a function
```

Reply:
BTW, `popup` seems to be used in the main message pane and menu is still shown in the top left corner. Let's wait a bit with the menu positioning for now, I'd better finish with other things (flatpak has to be done properly)